climateconnect/climateconnect
# Fix: Organization names that are too long cut info on preview

## Description
Closes #1099 

Introduce truncation to organization names longer than 2 lines.

## Test plan

1. Create an organzation or edit one and set the name to something very long
2. Go to /browse to see organization previews.
3. The name should be truncated
4. Also see same change for landing page organization previews

## Before landing

1. PR has meaningful title
1. `yarn lint` passes (frontend)
1. `yarn format` passes (frontend)
1. `make format` passes (backend)
